CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ The following is a curated list of changes in the Enact sandstone module, newest
 ### Fixed
 
 - `sandstone/Steps` prop `size` to accept number type
+- `sandstone/ContextualPopupDecorator` to update its position when the window size has been resized
```

Comment:
Please sync your branch with the latest develop.
And put the change log in alphabetical order under `Fixed` category.

Reply:
Thanks, Modified